components/Flashcard/Flashcard.js

```diff
@@ -354,6 +371,8 @@ export default function Flashcard({
                         variant={
                           isCorrect ? "markAsIncorrect" : "markAsCorrect"
                         }
+                        disabled={!session}
+                        grayout={!session}
```

Comment:
please update this for every occurrence to use only disabled for both internal props.

Reply:
see above